src/core/data-tests.js

```diff
@@ -23,11 +23,39 @@ export const name = "core/data-tests";
 const lang = defaultLang in l10n ? defaultLang : "en";
 
 function toListItem(href) {
-  return hyperHTML.bind(document.createElement("li"))`
+  
+  // const modifiedHref = href.split("/").pop().split(".")[0];
+  
+  if ( href.split("/").pop().split(".").includes("html") ||  href.split("/").pop().split(".").includes("htm") ) {
```

Comment:
Nit: make this two steps, and combine the search (otherwise, you are doing things twice):
```
// the last item will be the filename
const [testFile] = new URL(href).pathname.split("/").reverse();

// next we want to work out the type
const testParts = testFile.split(".");
const [testFile] = testParts; // the first item 
const isSecureTest = testParts.find(part => part === "https");
// this is probably incorrect, but find "manual" like this:  
const isManualTest = testFile.split("-").find(part => part === "manual");

// then 
if(isSecure){
  // add secure emoji...
}

if(isManualTest) {
  // add manual emoji
} 

```